src/System.IO.FileSystem.AccessControl/src/System/IO/FileSystemAclExtensions.netcoreapp.cs

```diff
@@ -24,5 +36,166 @@ public static void Create(this DirectoryInfo directoryInfo, DirectorySecurity di
 
             FileSystem.CreateDirectory(directoryInfo.FullName, directorySecurity.GetSecurityDescriptorBinaryForm());
         }
+
+        /// <summary>
```

Comment:
Let me know if you agree with the documentation.

Reply:
Did you refer to the constructor that this was meant to replace?

Reply:
I can add it to the remarks. You mean the [old Framework constructor for FileStream](https://referencesource.microsoft.com/#mscorlib/system/io/filestream.cs,564), right?

Reply:
Yeah I think you do. I will also add a similar remark to the DirectoryInfo Create method.